### DISCUSSION TITLE
Welcome to spacebar Discussions!

### DISCUSSION THREAD

**Discussion by cmacrae** [cmacrae/spacebar#64] [General]
## 👋 Welcome
This is a place to discuss all-things-spacebar! Got a general question about spacebar and its configuration/setup? This is the place for it.  
Want to share ideas, or setups - perhaps show off your latest style? Show others here!

Discussions is a place to connect with other members of the community, so be sure to:
  * Ask questions you’re wondering about 🤔 
  * Share ideas 💡 
  * Engage with others 🧑‍🤝‍🧑 
  * Welcome new-comers 🙋

**Comment by vimburton** [comment]
Hi, big thanks for this great project. 

Love the simple and minimalist features. I want it looking like a no BS linux wm, no widgets. Find it more suitable for my needs than the other alternatives!

Your browser in the readme looks cool. Is it firefox with userCrome tweaks?
